src/control/cmd/daos_agent/mgmt_rpc.go

```diff
@@ -109,13 +111,20 @@ func (mod *mgmtModule) handleGetAttachInfo(ctx context.Context, reqb []byte, pid
 		return nil, drpc.UnmarshalingPayloadFailure()
 	}
 
-	mod.log.Debugf("GetAttachInfo req from client: %+v", pbReq)
+	client := &procInfo{
+		pid: pid,
+	}
+	procName, err := common.GetProcName(int(pid))
+	if err == nil {
+		client.name = procName
+	}
+	mod.log.Tracef("%s: %s", client, pblog.Debug(pbReq))
```

Comment:
IMO this should stay at debug.

Reply:
Please remember that these will be happening for every single application connection. The balance that I'm trying to strike with these agent logs in particular is allowing DEBUG to be turned on at scale without causing a complete avalanche of log entries. The cached `GetAttachInfoResp` is logged once near the beginning of the log and doesn't change, so I'm not sure why we would need to log it for every application connection in the normal debug case.

Please compare the agent logs for a "busy" test, if you haven't already:
- [IOR small (new)](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12017/4/artifact/Functional%20Hardware%20Medium/ior/small.py/daos_logs.wolf-253/test_ior_small_daos_agent.log)
- [IOR small (old)](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/master/8427/artifact/Functional%20Hardware%20Medium/ior/small.py/daos_logs.wolf-25/test_ior_small_daos_agent.log)

The new-style agent logs should be even more compact (but still very useful!) once [DAOS-13151](https://daosio.atlassian.net/browse/DAOS-13151) has been addressed.

Reply:
Fair point, as long as it's logged somewhere without turning on Trace.